docusaurus.config.js

```diff
@@ -118,7 +118,8 @@ const config = {
         darkTheme: darkCodeTheme,
       },
       algolia: {
-        apiKey: '3f4f2f0325144237da8308f56a43e48d',
+        appid: 'UAFD8IBIF7',
```

Comment:
```suggestion
        appId: 'UAFD8IBIF7',
```

Looking at the preview, it still query the old appId